bottlerocket/samples/Makefile.toml

```
@@ -9,7 +9,13 @@ CLUSTER_TYPE = { value = "eks", condition = { env_not_set = ["CLUSTER_TYPE"] } }
 ASSUME_ROLE = { value = "~", condition = { env_not_set = ["ASSUME_ROLE"] } }
 AWS_REGION = { value = "us-west-2", condition = { env_not_set = ["AWS_REGION"] } }
 UPGRADE_VERSION = { value = "v1.11.1", condition = { env_not_set = ["UPGRADE_VERSION"] } }
-STARTING_VERSION = { value = "v1.11.0", condition = { env_not_set = ["STARTING_VERSION"] } }
+STARTING_VERSION = { script = ["""\
+        aws ssm get-parameter \
+            --region us-west-2 \
+            --name "/aws/service/bottlerocket/aws-k8s-1.24/arm64/latest/image_version" \
+            --query Parameter.Value \
+        | sed -rn 's/^"([0-9]+\\.[0-9]+\\.[0-9]+)-.*/v\\1/p' \
```

```suggestion
        aws ssm get-parameter \
            --region us-west-2 \
            --name "/aws/service/bottlerocket/aws-k8s-1.24/arm64/latest/image_version" \
            --query Parameter.Value \
            --output text
        | awk -F- '{printf "v%s", $1}' \
```

What you have isn't wrong, but this might be easier to read.